### PR TITLE
(SIMP-3335) Updated rsync build dir to rsync_skeleton

### PR DESCRIPTION
**Pull request by nick-markowski**
SIMP-3335 #comment helpers updated

### CODE REVIEW

File: lib/simp/rake/build/pkg.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@ def define_tasks
           @build_dirs = {
             :modules => get_module_dirs(args[:method]),
             :aux => [
-              "#{@src_dir}/rsync",
+              "#{@src_dir}/rsync_skeleton",
```
Comment (jeefberkey) [inline lib/simp/rake/build/pkg.rb]: Can it check for both, just in case? We should still prefer the new location.

Comment (nick-markowski) [inline lib/simp/rake/build/pkg.rb]: Actually, we need to for the gem to be backwards-compatible. Good call.

File: lib/simp/rake/build/pkg.rb [hunk 2]
```diff
@@ -46,7 +46,9 @@ def define_tasks
           @build_dirs = {
             :modules => get_module_dirs(args[:method]),
             :aux => [
+              # Legacy rsync, keep it for backwards-compatibility
               "#{@src_dir}/rsync",
+              "#{@src_dir}/rsync_skeleton",
```
Comment (trevor-vaughan) [inline lib/simp/rake/build/pkg.rb]: This shouldn't be called 'rsync_skeleton' on the filesystem.

The repo can be 'rsync_skeleton' but the directory should just be 'rsync', just like the 'environment_skeleton'

**Comment by trevor-vaughan** [comment]
Reverted the base directory to just `rsync` but kept the source as `rsync_skeleton`. Closing.

**Pull request closed without merging**